docs/basics/101-116-sharelocal.rst

```diff
@@ -263,40 +265,94 @@ explore the file hierarchy. The file content, however, is not present yet.
+   (such as ``recordings/longnow``), :command:`get` will operate on it to 1) install
+   it -- if it is a not yet installed subdataset -- and 2) retrieve the contents of any files.
+   That makes it very easy to get your file content, regardless of
+   how your dataset may be structured -- it is always the same command, and DataLad
+   blurs the boundaries between superdatasets and subdatasets.
+
+   In the above example, we called :command:`datalad get` with the option ``-n/--no-data``.
+   This option prevents that :command:`get` obtains the data of individual files or
+   directories, thus limiting its scope to the level of datasets as only a
+   :command:`datalad clone` is performed. Without this option, the command would
+   have retrieved all of the subdatasets contents right away. But with ``-n/--no-data``,
+   it only installed the subdataset to retrieve the meta data about file availability.
+
+To explicitly install all potential subdatasets *recursively*, that is,
+all of the subdatasets inside it as well, one can give the
+``-r``/``--recursive`` option to :command:`get`::
+
+  datalad get -n -r <subds>
+
+This would install the ``subds`` subdataset and all potential further
+subdatasets inside of it, and the meta data about file hierarchies would
+have been available right away for every subdataset inside of ``subds``.
+
+So why is a recursive get not the default behavior?
+In :ref:`nesting` we learned that datasets can be nested *arbitrarily* deep.
+Upon getting the meta data of one dataset you might not want to also install
+a few dozen levels of nested subdatasets right away.
 
-To explicitly install a dataset right away
-*recursively*, that is, all of the subdatasets inside it as well, one
-has to specify the ``-r``/``--recursive`` option::
+However, there is a middle way: The ``--recursion-limit`` option let's
+you specify how many levels of subdatasets should be installed together
+with the first subdataset::
 
-  datalad install --source ../DataLad-101 -r --description "DataLad-101 in mock_user"
+  datalad get -n -r --recursion-limit 1 <subds>
 
-would have installed the ``longnow`` subdataset as well, and the meta
-data about file hierarchies would have been available right from the
-start.
 
-So why is this behavior disabled by default?
-In :ref:`nesting` we learned that datasets can be nested *arbitrarily* deep.
-Upon installing a dataset you might not want to also install a few dozen levels of
-nested subdatasets right away.
+.. findoutmore:: datalad clone versus datalad install
 
-However, there is a middle way: The ``--recursion-limit`` option let's
-you specify how many levels of subdatasets should be installed together
-with the superdataset::
+   You may remember from section :ref:`installds` that DataLad has two commands to obtain datasets,
+   :command:`datalad clone` and :command:`datalad install`. They look and
+   feel surprisingly similar, and seem to result in identical outcomes.
```

Comment:
Nit pick: I guess for the reader it is not that surprising, because they only know `clone` so far. And that similarity might also be relatively short-lived with the transition of `clone` into the core API.